tests/core/test_ensemble.py

```diff
@@ -87,6 +94,34 @@ def test_policy_priority():
     assert result == priority_2_result
 
 
+def test_fallback_mapping_restart():
+    domain = Domain.load("data/test_domains/default.yml")
+    events = [
+        ActionExecuted(ACTION_DEFAULT_FALLBACK_NAME),
+        user_uttered(USER_INTENT_RESTART, 1),
```

Comment:
doesn't change the result of the test, but this should be the right history (fallback gets executed, we are waiting for user input, user gives `/restart` intent)
```suggestion
        ActionExecuted(ACTION_LISTEN),
        user_uttered(USER_INTENT_RESTART, 1),
```

Reply:
I left this as is since discussion means there won't necessarily be an `ACTION_LISTEN`